src/pipx/constants.py

```diff
@@ -4,20 +4,31 @@
 from pathlib import Path
 from textwrap import dedent
 from typing import NewType, Optional
+from platformdirs import user_cache_path, user_data_path, user_log_path
```

Comment:
I'm sure this will break the style guide.. Can you run `isort` or another tool on that?

Reply:
Both isort and black just add a new line before line 7.